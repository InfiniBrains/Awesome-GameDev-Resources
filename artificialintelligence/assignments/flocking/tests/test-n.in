100.000 100.000 100.000 100.000 0.000 0.000 0.000 5
1.234 5.678 0.000 0.000
4.321 8.765 0.000 0.000
5.678 1.234 0.000 0.000
8.765 4.321 0.000 0.000
1.234 4.321 0.000 0.000
0.010
0.100
0.001
1.000
321.123
