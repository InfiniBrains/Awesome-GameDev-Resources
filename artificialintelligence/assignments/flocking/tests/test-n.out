1.234 5.678 0.000 0.000
4.321 8.765 0.000 0.000
5.678 1.234 0.000 0.000
8.765 4.321 0.000 0.000
1.234 4.321 0.000 0.000
1.234 5.678 0.000 0.000
4.321 8.765 0.000 0.000
5.678 1.234 0.000 0.000
8.765 4.321 0.000 0.000
1.234 4.321 0.000 0.000
1.234 5.678 0.000 0.000
4.321 8.765 0.000 0.000
5.678 1.234 0.000 0.000
8.765 4.321 0.000 0.000
1.234 4.321 0.000 0.000
1.234 5.678 0.000 0.000
4.321 8.765 0.000 0.000
5.678 1.234 0.000 0.000
8.765 4.321 0.000 0.000
1.234 4.321 0.000 0.000
1.234 5.678 0.000 0.000
4.321 8.765 0.000 0.000
5.678 1.234 0.000 0.000
8.765 4.321 0.000 0.000
1.234 4.321 0.000 0.000
